packages/accounts-controller/src/AccountsController.test.ts

```diff
@@ -1043,6 +1082,7 @@ describe('AccountsController', () => {
       });
 
       it('delete the account and select the account with the most recent lastSelected', async () => {
+        const currentTime = Date.now();
```

Comment:
Use the new helper here too!

Reply:
I think we can skip it here in the test.